cmd/notary/tuf.go

```diff
@@ -357,11 +359,23 @@ func (ps passwordStore) Basic(u *url.URL) (string, string) {
 }
 
 func getTransport(gun string, readOnly bool) http.RoundTripper {
+	// Attempt to get a root CA from the config file. Nil is the host defaults.
+	rootPool := x509.NewCertPool()
+	rootCAFile := viper.GetString("remote_server.root_ca")
+	if rootCAFile != "" {
+		rootCert, err := trustmanager.LoadCertFromFile(viper.GetString("remote_server.root_ca"))
```

Comment:
You can use `rootCAFile` here. Also, https://github.com/docker/notary/pull/210#issuecomment-147118693